src/components/books/Books.js

```diff
@@ -26,30 +31,10 @@ function Books() {
     <div>
       <h2>Books</h2>
       <ul>
-        {books.map((book) => (
-          <div className="" key={book.id}>
-            <li key={book.id}>{book.title}</li>
-            <li>{book.Authur}</li>
-            <li>{book.completed ? 'Completed' : 'Not Completed'}</li>
-            <button type="button">Remove</button>
-          </div>
-        ))}
+        <Book books={books} />
```

Comment:
[OPTIONAL] I think our `<Book />` component should represent a single book in the DOM. What I mean by this is we should map through our array of books in this component and pass only the necessary pieces of information to the `<Book />` component to render one single book.

This will help you keep the code semantic and organized.